auditbeat/module/file_integrity/event_test.go

```diff
@@ -312,6 +316,45 @@ func TestBuildEvent(t *testing.T) {
 		assertHasKey(t, fields, "hash.sha1")
 		assertHasKey(t, fields, "hash.sha256")
```

Comment:
hash isn't supposed to be at the top level.  It should be file.hash.sha1 & file.hash.sha256. Also hashes should be added to "related.hash"

Reply:
Good catch, but I'll have to duplicate the `hash` entries since they are currently top-level in the data set since otherwise it's a breaking change. But I'll make a note to strip out the top-level fields in `8.x`.